hdf/src/hdfi.h

```diff
@@ -88,6 +88,10 @@
 #include <limits.h>
 #include <string.h>
 
+/* Temporarily eliminate implicit declaration of funcion 'yyerror' */
+int yylex(void);
+void yyerror(char* s);
```

Comment:
It looks like this is from the parser generator? Usually we can stick this in the generator script.

Reply:
This can go in a %top directive in the source .y/.l file(s)